CMake/setup_KWIVER.sh.in

```diff
@@ -7,6 +7,5 @@ export VG_PLUGIN_PATH=$this_dir
 export PATH=$this_dir/bin:$PATH
 export LD_LIBRARY_PATH=$this_dir/lib:$LD_LIBRARY_PATH
 export KWIVER_PLUGIN_PATH=$this_dir/lib/modules:$KWIVER_PLUGIN_PATH
```

Comment:
somewhat unrelated to this branch, but why is the KWIVER plugin install path "lib/modules" and not "lib/kwiver" "lib/arrows"?  "modules" is too generic and doesn't indicated that these are the KWIVER modules.  It could easily conflict with some other system package.  I see that Sprokit plugins install to "lib/sprokit".  Maybe it should be "lib/kwiver/arrows" and "lib/kwiver/sprokit"?  Thoughts?